src/routes/resources/resources.handlers.ts

```
@@ -52,25 +52,44 @@ export const getAll: AppRouteHandler<GetAllRoute> = async (c) => {
   }
 
-  // Get filtered resources
+  // Build where clause
+  const baseWhere = and(
+    ...(resourceType && isResourceType(resourceType)
+      ? [eq(resources.resourceType, resourceType)]
+      : []),
+    ...(categoryName ? [eq(categories.name, categoryName)] : []),
+    ...(tags.length > 0 ? [inArray(resourceTags.name, tags)] : []),
+    ...(cursor ? [lt(resources.id, cursor)] : [])
+  );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)